spec/policy/rate_limit/rate_limit_spec.lua

```diff
@@ -234,6 +234,7 @@ describe('Rate limit policy', function()
       assert.equal('10', window)
 
       os.execute("sleep 1")
```

Comment:
Haven't noticed this earlier. There is `ngx.sleep()` that is better than spawning a subprocess.